gabrielaEnache/src/main/java/oop/store/Item.java

```diff
@@ -0,0 +1,60 @@
+package oop.store;
+
+public class Item {
+    private static long idCounter = 0;
+    private long id;
```

Comment:
I suppose 'id' should be final, it can be generated through UUID

Reply:
Changed to final.
Thank you for suggestion but isn't the only right approach.